docs/src/index.md

```diff
@@ -1,17 +1,17 @@
 # ExaPF
 
 [`ExaPF.jl`](https://github.com/exanauts/ExaPF.jl) is a
-package to solve the power flow problem on upcoming exascale architectures by solving a system of nonlinear equations and provide derivative information used for example in a reduced space optimization method.
```

Comment:
is a differentiable power flow solver providing up to second-order derivatives? I know you mention it below. But for those who read only one sentence.

Reply:
indeed, good suggestion